frontend/packages/core/src/Feedback/stories/hint.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import type { Meta } from "@storybook/react";
+
+import Hint from "../hint";
+
+export default {
+  title: "Core/Feedback/Hint",
+  component: Hint,
+} as Meta;
+
+const Template = (props: { url: string; text: string }) => (
+  <Hint>
+    {props.url && <img src={props.url} />}
+    <div style={{ padding: "10px" }}>{props.text}</div>
```

Comment:
shouldn't the padding be part of the component?

Reply:
The padding is only there because of the image component. In a real world application the elements passed into the Hint would have their own styling which is what this is attempting to demonstrate